content/en/apps/_index.md

```diff
@@ -4,20 +4,13 @@ linkTitle: "CHT Applications"
 identifier: "apps"
 weight: 2
 description: >
-  Overview and reference for building digital health apps
+  Overview and reference for building CHT applications
 ---
 
-{{% pageinfo %}}
-This section provides an overview and reference for building digital health applications with CHT Core.
-{{% /pageinfo %}}
+To create a digital health tool using the Community Health Toolkit: build a CHT Application or re-use an existing CHT Application. The [CHT Core Framework]({{ ref="core" }}) provides a foundation on which many digital health tools can be conceptualised and built. A CHT Application takes that foundation and builds on it to create a concrete digital health tool tailored to the health workflows in a targeted community. 
 
-From a technical perspective, developing a custom app begins with writing XForms, JSON, and JavaScript code that configures the Core Framework’s features to meet your organization’s needs. 
+Since all CHT Applications share the same foundation, they also share capabilities and attributes. For example, all CHT Applications share a similar user-interface and share the same look-and-feel. All CHT Applications can be built to scale, can run across a variety of hardware, support offline-first experiences, and support most languages. These baseline capabilities are universal across all CHT Applications. Learn more about this in [Why the CHT?]({{ ref="why-the-cht" }})
```

Comment:
```suggestion
Since all CHT Applications share the same foundation, they also share capabilities and attributes. For example, all CHT Applications share a similar user-interface and share the same look-and-feel. All CHT Applications can be built to scale, can run across a variety of hardware, support offline-first experiences, and support most languages. These baseline capabilities are universal across all CHT Applications. Learn more about this in [Why the CHT?]({{< ref="why-the-cht" >}})
```